content/deploy/cdp_public.adoc

```diff
@@ -0,0 +1,82 @@
+
+What is important to note here is that we attempt to allow the user to define the least amount of information necessary to produce the target Deployment, something sometimes refered to as Goal Based Declarative Deployment. You can set as many overrides as you like, but you then take on responsibility for knowing how they will interact, whereas if you use the defaults they are already known to be best-practice.
+
+==== Infrastructure
+
+Controlled under the ‘infra’ key in all the underlying cloud infrastructure services you could need, most defaults are either in the https://github.com/cloudera-labs/cloudera.exe/blob/main/roles/common/defaults/main.yml[cloudera.exe.common] or https://github.com/cloudera-labs/cloudera.exe/blob/main/roles/infrastructure/defaults/main.yml[cloudera.exe.infra] defaults files. It is worth examining these defaults if you are interested in which subnet CIDRs, storage types, and other values are used in deployment.
+
+If you include the `infra` key in your definition without anything else, cloudera-deploy will create a base set of infrastructure (network, storage, security groups, etc.) on your chosen provider when you use one of the deployment triggering tags.
+
+.e.g. To create Infrastructure for CDP Public on Azure:
+[source,yaml]
+infra_type: azure
+infra:
+
+==== CDP Public Environment & Datalake
+
+This is the core CDP Public ‘platform’ to which the `plat` Ansible Tag refers, and is controlled by the top-level yaml key `env`. Apart from the common defaults mentioned earlier, there are platform specific defaults as well in https://github.com/cloudera-labs/cloudera.exe/blob/main/roles/platform/defaults/main.yml[cloudera.exe.platform].
+
+As with other top level keys, if you include `env:` on its own, then cloudera-deploy will generate an infrastructure suitable for CDP Public and Private based on your Definition, Profile, and other defaults.
+
+There are a large number of defaults in this role, you are advised to scroll through the defaults file above - we resist listing them all here to avoid additional maintenance burden.
+
+.e.g. To create a CDP Public Infrastructure on GCP with a Datalake pinned to a particular version:
+[source,yaml]
+infra_type: gcp
+env:
+  datalake:
+    version: 7.2.12
+
+==== Datahubs
+
+`datahub:` is the next key we look at, and the first that requires some child keys to work.
+
+As with env, if you have datahub in your definition file cloudera-deploy will ensure that the requisite Infrastructure and CDP Public Platform are deployed.
+
+For datahubs, you are required to submit a list of configurations, one for each datahub you want to be present. You have three options; you may give the name of a definition to deploy it from defaults, you may refer to a jinja template, or you may explicitly set the necessary values.
```

Comment:
colon after "three options"

Reply:
I think it would be better to have a bullet list with the three options.
It would be also good to have examples of each option's syntax?